pyuvdata/uvdata.py

```diff
@@ -2069,7 +2252,7 @@ def read_uvh5(self, filename, antenna_nums=None, antenna_names=None,
             if not read_data and len(filename) > 1:
                 raise ValueError('read_data cannot be False for a list of uvh5 files')
 
-            self.read_uvh5(filename[0], antenna_nums=antenna_nums,
+            self.read_uvh5(filename[0], axis=axis, antenna_nums=antenna_nums,
```

Comment:
Why is axis getting passed in here (and on line 2267)? I don't think it changes anything, but it's inconsistent with the other read functions.

Reply:
you're right, I'm not sure why it's there, I'll take it out.